sdk/eventhub/azure-eventhub/azure/eventhub/_pyamqp/aio/_session_async.py

```diff
@@ -288,7 +296,11 @@ async def _incoming_transfer(self, frame):
         try:
             await self._input_handles[frame[0]]._incoming_transfer(frame)  # handle
         except KeyError:
-            pass  #TODO: "unattached handle"
+            await self._set_state(SessionState.DISCARDING)
+            await self._connection.close(error=AMQPSessionError(
```

Comment:
Same comments as above - we shouldn't be passing in an exception here. Additionally this is a link-level error, so I'm not sure it should result in the connection being closed.